app/src/processing/app/platform/LinuxPlatform.java

```diff
@@ -40,20 +40,6 @@
   public void initBase(Base base) {
     super.initBase(base);
 
-    String javaVendor = System.getProperty("java.vendor");
```

Comment:
Could consider leaving a form if this back in if desired. Leave that for other reviewers to decide.